test/runtests.jl

```diff
@@ -79,7 +83,9 @@ function validkeys(node::IntervalTrees.LeafNode, minint, maxint)
             return false
         end
 
-        if !(minint <= entry <= maxint) || last(entry) > node.maxend
+        interval = IntervalTrees.Interval(first(entry), last(entry))
+
+        if !(minint <= interval <= maxint) || last(interval) > node.maxend
```

Comment:
Please review this code in light of #36. While this is just a workaround a complete elaborate fix will needed as described in #36.